bin/wp-cli.php

```diff
@@ -171,6 +175,18 @@ public function index( $args, $assoc_args ) {
 			if ( $assoc_args['limit'] && $assoc_args['limit'] < $assoc_args['bulk'] )
 				$assoc_args['bulk'] = $assoc_args['limit'];
 
+			if ( isset( $assoc_args['from'] ) || isset( $assoc_args['to'] ) ) {
+				global $searchpress_cli_date_range;
```

Comment:
revert to how you were doing this before. move the info to a class property, and for your callback method for the filter, start the method name with two underscores. wp-cli will ignore all methods starting with `__` and won't turn them into subcommands.